src/hooks/useChartDomain.js

```diff
@@ -0,0 +1,20 @@
+import { useMemo } from 'react';
+
+export default function useChartDomain(data, dataKeys, fallback = [0, 100], percentage = 10) {
+  const values = useMemo(() => {
+    if (!data || !data.length || !dataKeys || !dataKeys.length) {
```

Comment:
interesting that recharts actually support domain value with 2 functions, not numbers, like:
```
[max => max * 0.75, min => min * 1.25]
```
inside functions you can do whatever you want.

the only issue it doesn't support multiple keys anyway

Reply:
Yeah 😀